### ISSUE TITLE
question about COCO dataset

### ISSUE THREAD

**Issue by fabrizioguillaro** [li-yanhao/masksim#2]
Hello, I have a question about the COCO subset used in your work.

Are the 5000 COCO images the 5000 from COCO 2017 validation set?
Or are they a mix of COCO 2017 training set and validation set?

**Comment by li-yanhao** [comment]
Hi. Yes it was the COCO 2017 validation set containing 5k images. 4200 images were used for training our model and 800 images for validation.

**Comment by fabrizioguillaro** [comment]
great, thank you!
